### PR TITLE
Reducing traffic to Memcache while importing new features

### PR DESCRIPTION
**Pull request by Stromij**
Adding a Clustering-Algorithm to the tank in the feature import process reduces the traffic to the Memcache. It clusters dynamically the imported features by analysing  the distance between existing clusters and the newly imported feature.  
For comparision there is another algorithm called Bounding. It calculates all updated Tiles by a newly imported Tiles. By collecting more than one feature in the list the traffic to the Memcache is reduced, but it is more RAM necessary than with the Cluster-Algorithm.

### CODE REVIEW

File: src/main/kotlin/io/marauder/tank/RegionManager.kt [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                    biggestDistance = 0.0
+                    shortestDistance = Double.MAX_VALUE
+                    regions.forEach {r->recalculateDistantces(r)}
+                }
+            }
+            else {
+                // Füge f in region mit Index index ein.
+                addInRegion(index, f)
+
+                // Füge jeder Regionen, die in diesem Schritt nicht bearbeitet wurde einen Schritt im Zähler hinzu
```
Comment (f7o) [inline src/main/kotlin/io/marauder/tank/RegionManager.kt]: Please all comments in English! :)

Comment (Stromij) [reply src/main/kotlin/io/marauder/tank/RegionManager.kt]: Here we go ;)

File: resources/application.conf [hunk 2]
```diff
@@ -19,8 +19,14 @@ ktor {
         memcached_client_host = ${?TANK_MEMCACHED_CLIENT_HOST}
         memcached_client_port = 11211
         memcached_client_port = ${?TANK_MEMCACHED_CLIENT_PORT}
-        memcached_enabled = false
+        memcached_enabled = true
```
Comment (f7o) [inline resources/application.conf]: `false` by default sounds more suitable to me, as i try to keep the "quick start" an easy setup

Comment (Stromij) [reply resources/application.conf]: Forgot to rechange this. Should've doublchecked this before....

**Pull request merged**